webapp/channels/src/components/rhs_thread/rhs_thread.tsx

```diff
@@ -39,7 +37,7 @@ const RhsThread = ({
         }
     }, [currentTeam, channel]);
 
-    if (posts == null || selected == null || !channel) {
+    if (selected == null || !channel) {
```

Comment:
That will be handled by whatever handles it now. `posts` can never be undefined currently because the selector for it always returns an array, so it was impossible for `posts == null` to be true